src/runner.rs

```diff
@@ -224,4 +226,29 @@ When your transaction is complete, enter 'commit' or 'abort' as appropriate."#,
         }
         Ok(())
     }
+
+    pub(crate) async fn handle_status(&self) -> Result<()> {
```

Comment:
I think this method is useful, but it really should be `/ping` and not `/status`. I think status should be something that doesn't need another request, e.g. the shell is *always* observing network latency etc. and asking for the status is just about returning it from memory. At a high level, we might be able to use `tracing` to implement that.

Reply:
Did we want to have both `ping` and `status` options for the user, or just `status`?
If we wanted both, then I would imagine `ping` only return:
* Connection status
* round-trip time

The other info such as:
* Current ledger
* Current region
* Current version 
* 
would be part of `status`


When you say `tracing`, are you referring to: crate [Tracing](https://docs.rs/tracing/0.1.26/tracing/)?

Reply:
Good suggestions, Byron!

I think what you're proposing is we (in this PR) add `/ping` that sends a ping request and prints the round trip time (and that's it). Then, you're also saying we (in this, or another PR) add `/status` that prints the ledger/region/version (but does not send a ping). In the future, `/status` might return latency information from recent commands if we're able to capture it.

yes re: tracing.

Reply:
Thanks for clarifying. I updated the PR to both `\ping` and `\status`. I added a #TODO comment in `handle_status()` to keep track of returning latency information without sending ping